script/fix-animation

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+import re
+import argparse
+
+
+def fix_animation(input: str, output: str) -> None:
+    with open(input) as f:
+        data = f.readlines()
+
+    output_data = []
+    for line in data:
+        if 'stylesheet' in line:
+            match = re.search(r'href="(.*?)"', line)
+            if match is None:
+                output_data.append(line)
+            else:
+                css_name = match[1].replace('\\', '/').split('/')[-1]
+                output_data.append(f'    <link rel="stylesheet" href="{css_name}">\n')
+        elif '<webots-view' in line:
+            match = re.search(
+                r'data-thumbnail=(.*?) .*data-scene=(.*?) .*data-animation=(.*?)>', line)
+            if match is None:
+                output_data.append(line)
+            else:
+                jpg_name = match[1].replace('\\', '/').split('/')[-1]
+                x3d_name = match[2].replace('\\', '/').split('/')[-1]
+                json_name = match[3].replace('\\', '/').split('/')[-1]
```

Comment:
Maybe worth asserting that the removed portion of these is all the same?
Perhaps like:
```suggestion
                *jpg_dir, jpg_name = match[1].replace('\\', '/').split('/')
                *x3d_dir, x3d_name = match[2].replace('\\', '/').split('/')
                *json_dir, json_name = match[3].replace('\\', '/').split('/')
                assert jpg_dir == x3d_dir == json_dir
```